net-p2p/retroshare/retroshare-0.6.5-r3.ebuild

```diff
@@ -0,0 +1,143 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
I see updating the old ebuild too, is there a reason to? i.e. are you planning to stabilize -r3 over 0.6.6 or something? Or is there a problem with 0.6.6 that will make users want to keep using 0.6.5?

I don't use this, so I wouldn't know.

Reply:
I plan to stabilize 0.6.6 but there might be problems with new version so I also wanted to do a minimal maintenance of 0.6.5 in case someone need to stick to that retroshare version